ftw/upgrade/command/utils.py

```diff
@@ -1,4 +1,7 @@
+from StringIO import StringIO
 from path import Path
+import sys
```

Comment:
Could you remove this import, as `sys` is already imported some lines below? :wink: :scissors: